search/tweets_engine.go

```diff
@@ -20,7 +23,7 @@ func (e *ESEngine) SearchTweets(query Query, sortBy string, from int, size int,
 			resultTypes:      []string{consts.ES_RESULT_TYPE_TWEETS},
 			index:            "",
 			query:            query,
-			sortBy:           sortBy,
+			sortBy:           consts.SORT_BY_RELEVANCE,
```

Comment:
add a comment:

`// Inside the carousel, the tweets are always sorted by relevance. The EffectiveDate of the carousel itself will be equal to the EffectiveDate of the most relevant tweet.`